src/migrations/user-store/file-name-migration.ts

```diff
@@ -6,12 +6,17 @@ export async function fileNameMigration() {
   const userDataPath = (app || remote.app).getPath("userData");
   const configJsonPath = path.join(userDataPath, "config.json");
   const lensUserStoreJsonPath = path.join(userDataPath, "lens-user-store.json");
-  const [configJsonExists, lensUserStoreJsonExists] = await Promise.all([
-    fse.pathExists(configJsonPath),
-    fse.pathExists(lensUserStoreJsonPath),
-  ]);
 
-  if (configJsonExists && !lensUserStoreJsonExists) {
+  try {
     await fse.move(configJsonPath, lensUserStoreJsonPath);
+  } catch (error) {
+    if (error.code === "ENOENT" && error.path === configJsonPath) { // (No such file or directory)
+      return; // file already moved
+    } else if (error.message === "dest already exists.") {
+      await fse.remove(configJsonPath);
```

Comment:
Assumes dest is newer (more relevant?) than `config.json`. Is this a safe assumption?

Reply:
That is indeed what my assumption is.

Some things I considered when coming to this conclusion:

1. The `UserStore` doesn't try and create the file until after the code has been called.
2. If we consider some developer that goes between versions we don't want to override the `dest` after a newer `src` has been made.

I guess we could try and look at the timestamps of last modified but that seems a bit too detailed.